src/adhocracy_mercator/adhocracy_mercator/sheets/test_mercator2.py

```diff
@@ -0,0 +1,778 @@
+from datetime import datetime
+from datetime import timedelta
+
+from pyramid import testing
+from pytest import fixture
+from pytest import mark
+from pytest import raises
+
+
+class TestUserInfoSheet:
+
+    @fixture
+    def meta(self):
+        from adhocracy_mercator.sheets.mercator2 import userinfo_meta
+        return userinfo_meta
+
+    @fixture
+    def context(self):
+        from adhocracy_core.interfaces import IItem
+        return testing.DummyResource(__provides__=IItem)
+
+    def test_create_valid(self, meta, context):
+        from zope.interface.verify import verifyObject
+        from adhocracy_core.interfaces import IResourceSheet
+        from adhocracy_mercator.sheets.mercator2 import IUserInfo
+        from adhocracy_mercator.sheets.mercator2 import UserInfoSchema
+        inst = meta.sheet_class(meta, context)
+        assert IResourceSheet.providedBy(inst)
+        assert verifyObject(IResourceSheet, inst)
+        assert inst.meta.isheet == IUserInfo
+        assert inst.meta.schema_class == UserInfoSchema
+
+    def test_get_empty(self, meta, context):
+        inst = meta.sheet_class(meta, context)
+        wanted = {'first_name': '',
+                  'last_name': '',
+                  }
+        assert inst.get() == wanted
+
+
+class TestOrganizationInfoSheet:
+
+    @fixture
+    def meta(self):
+        from adhocracy_mercator.sheets.mercator2 import organizationinfo_meta
+        return organizationinfo_meta
+
+    @fixture
+    def context(self):
+        from adhocracy_core.interfaces import IItem
+        return testing.DummyResource(__provides__=IItem)
+
+    def test_create_valid(self, meta, context):
+        from zope.interface.verify import verifyObject
+        from adhocracy_core.interfaces import IResourceSheet
+        from adhocracy_mercator.sheets.mercator2 import IOrganizationInfo
+        from adhocracy_mercator.sheets.mercator2 import OrganizationInfoSchema
+        inst = meta.sheet_class(meta, context)
+        assert IResourceSheet.providedBy(inst)
+        assert verifyObject(IResourceSheet, inst)
```

Comment:
Test to verify  the IResouceSheet  interface is not necessary if we use the default sheet class (adhocracy_core.sheets.AnnotationSheet)